### DISCUSSION TITLE
[1292220]Apoc Runner {仅本体}

### DISCUSSION THREAD

**Discussion by Kiowx** [pjy612/SteamManifestCache#781] [Share]
### Steam 游戏ID

1292220

### 密钥列表

```hash
1292221;7f97e4697f164ee21cc78ceb52dea48430f4f58d73fe1135bfdba52ba72a5b04
```

### 清单附件

[1292220.zip](https://github.com/user-attachments/files/16756298/1292220.zip)

### 备注

code:B1B73FBF770021F4
email:813463830@qq.com

**Comment by pjy612** [comment]
* 密钥格式
* #26

**Reply by Kiowx** [reply]
好了

**Reply by Kiowx** [reply]
现在还可以加入内测吗
